### PR TITLE
add more tables in the event-calendar view

### PR DESCRIPTION
**Pull request by LukeStorry**

### CODE REVIEW

File: src/site/views/event/tmpl/default.php [hunk 1]
```diff
@@ -101,7 +102,73 @@
...
+								echo "</tr>\n";
+							}
+							?>
+							</tbody>
+						</table>
+
+						<?php
+						  // Only want to show the full attendees table when the club committees actually need it
+						  $registrationClosed = time() > (24 * 60 * 60 + strtotime($item->event_date_close));
+							$eventFinished = time() > (72 * 60 * 60 + strtotime($item->event_date));
```
Comment (oshotton) [inline src/site/views/event/tmpl/default.php]: Interesting indentation here.

File: src/site/views/event/tmpl/default.php [hunk 2]
```diff
@@ -101,7 +102,73 @@
...
+								echo "<td>{$uni}</td>";
+								echo "<td>{$count}</td>";
+								echo "</tr>\n";
+							}
+							?>
+							</tbody>
+						</table>
+
+						<?php
+						  // Only want to show the full attendees table when the club committees actually need it
```
Comment (oshotton) [inline src/site/views/event/tmpl/default.php]: And when is it that they actually need it? Be good to say this explicitly.

File: src/site/views/event/tmpl/default.php [hunk 3]
```diff
@@ -101,7 +102,73 @@
...
+						  $registrationClosed = time() > (24 * 60 * 60 + strtotime($item->event_date_close));
```
Comment (oshotton) [inline src/site/views/event/tmpl/default.php]: Looks like 1 day after the ticket sales close. I would say this should be available from when ticket sales open to the end of the event. For some events, tickets are still being sold on the Friday of and President pack often released before ticket sales close. Would be too late to buy food and plan which houses everyone will be in etc.

**Comment by oshotton** [comment]
These tables might start getting quite big. Might be nice to put them in Bootstrap [Tabs](https://getbootstrap.com/docs/3.3/javascript/#tabs) or [Accordion](https://getbootstrap.com/docs/3.3/javascript/#collapse-example-accordion). Although this won't stop be approving this PR - can be done as an enhancement afterwards.

File: src/site/views/event/tmpl/default.php (hunk 1 above)
Comment (LukeStorry) [reply src/site/views/event/tmpl/default.php]: unsure how that happened, sorry

Comment (LukeStorry) [reply src/site/views/event/tmpl/default.php]: indentation looks fine on my editor, hmm

File: src/site/views/event/tmpl/default.php (hunk 3 above)
Comment (LukeStorry) [reply src/site/views/event/tmpl/default.php]: Yeah I was unsure on how useful these timings would be. I'll change to viewing from ticket open.

File: src/site/views/event/tmpl/default.php (hunk 2 above)
Comment (LukeStorry) [reply src/site/views/event/tmpl/default.php]: the date of event ticket sales opening isn't exposed here, so I'll just leave this if statement open-ended at the start, and if there are no tickets it'll just revert to "There is no ticket information to show at this time."

Comment (oshotton) [reply src/site/views/event/tmpl/default.php]: True, would need to add `event.event_date_open` to `getItem()` or join on the `event` table in `getEventAttendees()`. Having "There is no ticket information to show at this time." works too. 👍

**Pull request merged**